pkg/controllers/provisioning/scheduling/suite_test.go

```diff
@@ -4597,6 +4597,115 @@ var _ = Context("Scheduling", func() {
 			}
 		})
 	})
+
+	Describe("Dynamic Resource Allocation (DRA)", func() {
```

Comment:
Let's also include a test to verify that we ignore pods which specify resource claim requests for an init container. You could use a `DescribeTable` to avoid duplication between the standard and init container tests.